k4MarlinWrapper/k4MarlinWrapper/converters/EDM4hep2Lcio.h

```diff
@@ -58,11 +53,11 @@ struct CollectionPairMappings;
 class EDM4hep2LcioTool : public AlgTool, virtual public IEDMConverter {
 public:
   EDM4hep2LcioTool(const std::string& type, const std::string& name, const IInterface* parent);
-  virtual ~EDM4hep2LcioTool();
-  virtual StatusCode initialize();
-  virtual StatusCode finalize();
+  ~EDM4hep2LcioTool() override = default;
```

Comment:
```suggestion
```
No need to declare a destructor just override it to default - 'the rule of zero' recommends relying on the compiler-generated one
Same in `Lcio2EDM4hepTool`

Reply:
Done